foodhub/tests/api/test_user.py

```diff
@@ -0,0 +1,67 @@
+from urllib import response
+import pytest
+from rest_framework.test import APIClient
+
+from accounts.models import Merchant, Item, Store, Order
+
+client=APIClient()
+
+@pytest.mark.django_db
+def test_register_user(client):
+    payload= dict(
+        password1="subh@2310",
+        password2="subh@2310",
+        email="subhnesh0@gmail.com",
+        first_name="Subhnesh",
+        last_name="Kumar"
+    )
+    response = client.post('/account/register/', payload)
+    data=response.data
+
+    assert data['first_name'] == payload['first_name']
+    assert response.status_code == 201
+
+@pytest.mark.django_db
+def test_login_user(user, client):
+    response = client.post('/account/login/', dict(email="subhnesh0@gmail.com", password="subh@2310"))
+    assert response.status_code == 200
+
+
+
+@pytest.mark.django_db
+def test_merchant_creation_api_endpoint(user):
```

Comment:
Also check for the creation of the instance in db.

Reply:
As per requirement i have created new tests folder and define two file there conftest.py and tests.py and written a test cases